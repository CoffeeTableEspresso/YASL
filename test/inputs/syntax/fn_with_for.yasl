
fn f(const s) {
    for let i = 0; i < len s; i += 1 {
        const char = s[i]
        echo char
    }
}

f('eleetminicoworoep')
