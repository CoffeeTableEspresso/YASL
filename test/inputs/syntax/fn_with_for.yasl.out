e
l
e
e
t
m
i
n
i
c
o
w
o
r
o
e
p
